src/main/java/quack/Ui.java

```diff
@@ -0,0 +1,252 @@
+package quack;
+
+import java.util.Scanner;
+
+import quack.exception.InvalidIndexException;
+import quack.exception.InvalidTaskTypeException;
+
+import quack.tasks.Task;
+
+/**
+ * This class is responsible for displaying any UI interface,
+ * for the Quack chatbot.
+ */
+public class Ui {
+    
+    /** String to print out the spacers between each command */
+    private String spacer = "-".repeat(65);
+    /** The name of the chatbot */
+    private String botName = "Quack";
+    /** The logo for Quack */
+    private String logo =
+        "________                       __    \n" +
+        "\\_____  \\  __ _______    ____ |  | __\n" +
+        " /  / \\  \\|  |  \\__  \\ _/ ___\\|  |/ /\n" +
+        "/   \\_/.  \\  |  // __ \\\\  \\___|    < \n" +
+        "\\_____\\ \\_/____/(____  /\\___  >__|_ \\ \n" +
+        "       \\__>          \\/     \\/     \\/\n";
+
+    /** Farewell message for Quack */
```

Comment:
I like how you gave comments for readers to understand the intent of each variable.